### DISCUSSION TITLE
Issues with Generating Sitemap.xml File in Platforms Starter Kit

### DISCUSSION THREAD

**Discussion by heypratik** [vercel/community#1891] [Help] [answered]
### Summary

In the Platforms Starter Kit, I am experimenting with the page folder, which includes _site/[site]/index.js, representing the subdomain or main domain page of users. However, I encountered a problem while attempting to create a sitemap.xml file within [site]/seo/sitemap.xml.jsx, as it resulted in a 404 error.

I conducted some tests and found that removing ".XML" from the file name would make the sitemap work as /sitemap, but it still did not function as sitemap.xml. Additionally, I tested adding sitemap.xml in the root pages, which worked correctly and served the sitemap as /sitemap.xml.

Therefore, I am perplexed as to why the sitemap.xml file does not work within the [site]/seo folder.

**Comment by steven-tey** [comment, marked as answer]
Hey @heypratik! You can easily generate a dynamic sitemap file for your Platforms project simply by defining a `sitemap.xml.tsx` file at the root of your pages filter and use `getServerSideProps to fetch the correct paths.

I still need to update it in the Platforms Starter Kit but here's an example that I'm doing with my side project Dub: https://github.com/steven-tey/dub/blob/main/pages/sitemap.xml.tsx

Hope this helps!

**Reply by heypratik** [reply]
It works! Thank you!
